### ISSUE TITLE
Custom Etcd quota backend in CAPA

### ISSUE THREAD

**Issue by T-Kukawka** [giantswarm/roadmap#2977]
The Custom Etcd quota backend allows to increase the Etcd keyspace size to prevent Etcd from crashing. Recovering from this type of crash requires manual intervention. We have this configuration available in Vintage, which is adjusted especially for big clusters.

There is CAPA value which we already adjusted, now we have to expose it so it can be adjusted for big clusters. Let's make this value in a way so giant swarm engineers can do it at hand but after consultations with customers

**Comment by T-Kukawka** [comment]
We will need help from @giantswarm/team-turtles as it should be probably first provider agnostic in cluster chart

**Comment by Rotfuks** [comment]
We will put the configuration in the new cluster chart in the global helm values - we should hide it behind an "enabled"-flag to make sure it's truely internal.

**Comment by nprokopic** [comment]
Current value in cluster-aws (as well as in cluster chart) for `quota-backend-bytes` is set to `8589934592`, i.e. 8 GiB, which is the officially suggested maximum in etcd, see [here](https://etcd.io/docs/v3.5/dev-guide/limit/#storage-size-limit):

> **Storage size limit**
The default storage size limit is 2 GiB, configurable with --quota-backend-bytes flag. 8 GiB is a suggested maximum size for normal environments and etcd warns at startup if the configured value exceeds it.

Few questions here for @T-Kukawka.

1. Do we really need more than 8GiB in some cases?
2. Is [this](https://github.com/giantswarm/aws-operator/blob/d79a73d1fddd2ed70f5ff6bf7d986d94e8dce2fd/service/internal/cloudconfig/tccpn.go#L509) the place where we're setting custom values in AWS vintage?
3. And most importantly, are you sure that we are setting custom values on any WC at the moment? I checked all AWS workload clusters and looked for `etcd.giantswarm.io/quota-backend-bytes` annotation on the Cluster CR, and I found none. So either I missed something while checking (like `--quota-backend-bytes` being configured in some other way), or we're not customising `--quota-backend-bytes` anywhere 😄
